src/gsi/calctends_no_ad.f90

```diff
@@ -210,7 +211,7 @@ subroutine calctends_no_ad(st,vp,t,p,mype,u_t,v_t,t_t,p_t,uvflag)
       end do                         
     end if
 
-    call turbl_ad(ges_prsi(1,1,1,it),ges_tv,ges_teta(1,1,1,it),&
+    if(use_pbl)call turbl_ad(ges_prsi(1,1,1,it),ges_tv,ges_teta(1,1,1,it),&
```

Comment:
@jderber-NOAA   If in calctends.f90,  there is no "use_pbl" to decide if its' forward sub (turbl) is called, I am wondering why the condition (if(use_pbl) is added?